_data/cryptocurrencies.yml

```diff
@@ -186,6 +186,13 @@ websites:
       software: Yes
       doc: https://support.coins.ph/hc/en-us/articles/202604924-What-is-two-factor-authentication-
 
+    - name: COINUT
```

Comment:
On the main page title it says "Coinut" and not "COINUT". Which spelling is the right one?

Reply:
Changed to Coinut in the new commit.